src/modules/shared/services/innovations.service.ts

```diff
@@ -353,10 +407,10 @@ export class InnovationsService extends CoreService {
       }))
     );
   }
-  
-  getInnovationSubmission(innovationId: string): Observable<InnovationSubmissionDTO> {
+
+  getInnovationSubmission(innovationId: string): Observable<{ submittedAllSections: boolean, submittedForNeedsAssessment: boolean }> {
     const url = new UrlModel(this.API_INNOVATIONS_URL).addPath('v1/:innovationId/submissions').setPathParams({ innovationId });
-    return this.http.get<InnovationSubmissionDTO>(url.buildUrl()).pipe(take(1), map(response => response));
+    return this.http.get<{ submittedAllSections: boolean, submittedForNeedsAssessment: boolean }>(url.buildUrl()).pipe(take(1), map(response => response));
```

Comment:
Didn't comment before when I saw removing and changing types. But I prefer having DTOs for API responses than this sort of types.

Then the method either uses the DTO if it doesn't make any changes or it has a different return type when it does do changes.

This approach has two issues in my opinion:
- it doesn't have one place to go when changing a DTO from the API
- it doesn't ensure that response only has these two fields
specially when they are the same at